test/e2e/kubelet_perf.go

```diff
@@ -242,7 +246,10 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 			itArg := testArg
 			podsPerNode := itArg.podsPerNode
 			name := fmt.Sprintf(
-				"for %d pods per node over %v", podsPerNode, monitoringTime)
+				"resource tracking for %d pods per node", podsPerNode)
+			if podsPerNode >= 35 {
```

Comment:
@jayunit100 There are 3 test cases - "podsPerNode = 0, 35, 100". We only care about the last 2 cases. However, a constant improves the readability. Thanks! :)